packages/ra-ui-materialui/src/input/ArrayInput.tsx

```diff
@@ -69,6 +76,23 @@ export const ArrayInput = ({
         validate: sanitizedValidate,
         ...rest,
     });
+    
+    const {
+        id,
+        input,
+        isRequired,
+        meta: { error, touched },
+    } = useInput({
```

Comment:
You don't need `useInput`, `useFieldArray` returns a `meta` which contains the `error` if any. See https://github.com/final-form/react-final-form-arrays#fieldarrayrenderprops